### PR TITLE
Add JITMs after user publishes a post and when user uploads an image

### PR DESCRIPTION
**Pull request by eliorivero**
- After user publishes a post, a JITM will be displayed:
    * if user doesn't have Stats enabled, it will prompt to activate it. Once it's done, it will prompt user to go to WordPress.com
    * if user has Stats enabled, it will prompt user to go to WordPress.com

- After user starts uploading an image, a JITM will be displayed:
    * if user doesn't have Photon enabled, it will prompt to activate it.

### CODE REVIEW

File: _inc/jetpack-jitm.js [hunk 1]
```diff
@@ -28,10 +30,11 @@
 
 	function initEvents() {
 
-		var module_slug, success_msg, fail_msg, hide_msg;
+		var module_slug, success_msg, fail_msg, hide_msg,
+		    $body = $( 'body' );
```
Comment (dereksmart) [inline _inc/jetpack-jitm.js]: This was indented with mixed spaces & tabs, which is why Travis is complaining

File: class.jetpack-jitm.php [hunk 2]
```diff
@@ -272,6 +305,43 @@ function editor_msg() {
 		}
 	}
 
+	/**
+	 * Display message in editor prompting user to enable stats.
+	 *
+	 * @since 3.9
+	 */
+	function stats_msg() {
+		if ( current_user_can( 'manage_options' ) ) {
```
Comment (dereksmart) [inline class.jetpack-jitm.php]: is this necessary given we check `current_user_can( 'jetpack_manage_modules' )` in `prepare_jitms()`?

**Comment by dereksmart** [comment]
When testing the flow with stats inactive, when I click to enable Jetpack Stats, the 2nd CTA to "Go to wpcom" disappears after a few seconds

**Comment by eliorivero** [comment]
Fixed in 4b13bbb JITM events now only apply to themselves.

**Comment by zinigor** [comment]
Read, tested, works good!

**Comment by jeherve** [comment]
I ran some tests and experienced a few issues:

1) I couldn't get the Photon message to appear in the Media modal:

screen shot 2016-01-12 at 9 54 44 pm

It appears in Media > New, though:

screen shot 2016-01-12 at 9 55 58 pm

2) Not directly related to this PR, but wouldn't we want `editor_msg` to appear in `post.php`, as long as the post isn't published, like when the user saves a draft and continues to edit their post later?

**Comment by zinigor** [comment]
@jeherve the message appears in the left column of the modal *after you start uploading an image*, please try doing that.

**Comment by jeherve** [comment]
That didn't work, unfortunately:

screen shot 2016-01-12 at 10 27 10 pm

**Pull request merged**